web/packages/teleterm/src/ui/AppUpdater/WidgetView.tsx

```diff
@@ -0,0 +1,286 @@
+    updateEvent.autoUpdatesStatus.source.kind === 'most-compatible'
+      ? updateEvent.autoUpdatesStatus.source.skippedManagingClusterUri
+      : '';
+
+  return (
+    <AvailableUpdate
+      version={updateEvent.update.version}
+      platform={props.platform}
+      description={description}
+      unreachableClusters={unreachableClusters}
+      skippedManagingClusterUri={skippedManagingClusterUri}
+      downloadHost={downloadBaseUrl}
+      onMore={props.onMore}
+      getClusterName={getClusterName}
+      primaryButton={
+        button ? { name: button.name, onClick: button.action } : undefined
+      }
+    />
+  );
+}
+
+function AvailableUpdate(props: {
+  version: string;
+  description: string | { Icon: ComponentType<IconProps>; text: string };
+  unreachableClusters: UnreachableCluster[];
+  skippedManagingClusterUri: string;
+  downloadHost: string;
+  platform: Platform;
+  onMore(): void;
+  getClusterName(clusterUri: RootClusterUri): string;
+  primaryButton?: {
+    name: string;
+    onClick(): void;
+  };
+}) {
+  const hasUnreachableClusters = !!props.unreachableClusters.length;
+  const isNonTeleportServer =
+    props.downloadHost && !isTeleportDownloadHost(props.downloadHost);
+
+  return (
+    // Mimics a neutral alert.
+    <Stack
+      justifyContent="space-between"
+      gap={1}
+      width="100%"
+      css={`
+        border: 1px solid ${props => props.theme.colors.text.disabled};
+        background: ${props => props.theme.colors.interactive.tonal.neutral[0]};
+      `}
+      borderRadius={3}
+      px={3}
+      py="12px"
+    >
+      <Flex width="100%" alignItems="center" justifyContent="space-between">
+        <Flex gap={1} alignItems="center" width="100%">
+          {props.platform === 'darwin' ? (
+            <img alt="App icon" height="50px" src={iconMac} />
+          ) : (
+            <img
+              alt="App icon"
+              height="43px"
+              style={{ marginRight: '4px' }}
+              src={iconWinLinux}
+            />
+          )}
+          <Stack gap={0}>
+            <Text bold>Teleport Connect {props.version}</Text>
+            {typeof props.description === 'object' ? (
+              <Flex gap={1}>
+                <props.description.Icon size="small" />
+                <P3>{props.description.text}</P3>
+              </Flex>
+            ) : (
+              <P3>{props.description}</P3>
+            )}
+          </Stack>
+        </Flex>
+        <Flex gap={2}>
+          {props.primaryButton && (
+            <ButtonPrimary size="small" onClick={props.primaryButton.onClick}>
+              {props.primaryButton.name}
+            </ButtonPrimary>
+          )}
+          <ButtonSecondary size="small" onClick={props.onMore}>
+            More
+          </ButtonSecondary>
+        </Flex>
+      </Flex>
+      {(hasUnreachableClusters ||
+        props.skippedManagingClusterUri ||
+        isNonTeleportServer) && (
+        <Stack gap={0} ml={1}>
```

Comment:
```suggestion
        <Stack gap={1} ml={1}>
```

Otherwise I think the children are a bit too close to each other?